guides/common/modules/ref_permissions-required-to-provision-hosts.adoc

```diff
@@ -0,0 +1,79 @@
+[id="permissions-required-to-provision-hosts_{context}"]
+= Permissions Required to Provision Hosts
+
+The following list provides an overview of the permissions a non-admin user requires to provision hosts.
+
+Compute resource:: view_compute_resources, create_compute_resources, destroy_compute_resources, power_compute_resources
++
+Required to provision bare metal hosts.
+
+Compute resource:: view_compute_resources_vms, create_compute_resources_vms, destroy_compute_resources_vms, power_compute_resources_vms
++
+Required to provision VM hosts.
+
+Location:: view_locations
+
+Subnet:: view_subnets
+
+Hostgroup:: view_hostgroups, create_hostgroups, edit_hostgroups, play_roles_on_hostgroup
+
+Organization:: view_organizations
+
+Compute profile:: view_compute_profiles
+
+Host:: view_hosts, create_hosts, edit_hosts, destroy_hosts, build_hosts, power_hosts, play_roles_on_host
+
+Host:: view_discovered_hosts, submit_discovered_hosts, auto_provision_discovered_hosts, provision_discovered_hosts, edit_discovered_hosts, destroy_discovered_hosts
++
+Required if the Discovery service is enabled.
+
+Activation Keys:: view_activation_keys
+
+Domain:: view_domains
+
+Environment:: view_environments
+
+Image:: view_images
+
+Medium:: view_media
+
+Operatingsystem:: view_operatingsystems
+
+Ptable:: view_ptables
+
+Provisioning template:: view_provisioning_templates
+
+Architecture:: view_architectures
+
+Parameter:: view_params, create_params, edit_params, destroy_params
+
+ifdef::katello[]
```

Comment:
Satellite and orcharhino also use the Katello plug-in; also applies to below

Reply:
My bad, I couldn't find the permissions in my testing instance, so I thought they only applied to ifdef::katello. But turns out the resource types have different names. Updated.